hops/serving.py

```diff
@@ -162,7 +162,7 @@ def _start_or_stop_serving_rest(serving_id, action):
 
 def create_or_update(artifact_path, serving_name, serving_type="TENSORFLOW", model_version=1,
                              batching_enabled = False, topic_name="CREATE",  num_partitions = 1, num_replicas = 1,
-                             instances = 1):
+                             instances = 1, kfserving=False):
```

Comment:
Can you add this parameter to the docstring too?

Reply:
Ah, I missed that. Done.